COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs

```diff
@@ -53,12 +48,7 @@ public partial class ParameterTable
         /// <summary>
         /// The <see cref="ParameterBaseRowViewModelComparer" />
         /// </summary>
-        private ParameterBaseRowViewModelComparer comparer = new();
-
-        /// <summary>
-        /// The sorted collection of <see cref="ParameterBaseRowViewModel" />
-        /// </summary>
-        private ReadOnlyObservableCollection<ParameterBaseRowViewModel> sortedCollection;

```

Comment:
The reason why I removed this one it's because even with the "Connect" method subscription, rows from sortedCollection were not updating. 

The ParameterTable component subscribes only to the first viewmodel's rows (because the OnInitialized method is invoked only once when navigating between views of the same type) - making it not possible to subscribe to subsequent view's rows.

I'll work on an update that binds the sorted collection outside the OnInitialized method 🤙